examples/hello-world/hello-pt-resnet/README.md

```diff
@@ -0,0 +1,18 @@
+# Hello PyTorch ResNet
```

Comment:
Should this be a hello-xxx example?

Reply:
keep is as hello-pt-resnet for now. The implementation approach is the same.